### ISSUE TITLE
Banner Rez Not Working

### ISSUE THREAD

**Issue by Manatikik** [WarEmu/WarBugs#13322]
Banner rez is not working on a sub-40 RP repeatedly. Works fine on other level 40 toons. He doesn't get the rez pop-up nor does he get the buff saying he was targetted by the skill.

**Comment by grunbag** [comment]
Where did you tried the ability ?

**Comment by Manatikik** [comment]
Tried it in Avelorn last night inside keep walls and again in Cinderfall today in PvE. 

Also can't use it to avoid Terror like you can with M4 in instances but I assume thats intended :D

**Comment by grunbag** [comment]
Yes it’s intended , I’ll check . It happens only with RP?

**Comment by Manatikik** [comment]
Yes, I've used banner rez plenty of times on other characters but it won't work on this specific RP when I try to rez him; like I said he is sub-40 so maybe something weird about that?

**Comment by Manatikik** [comment]
Update: he has now hit 40 and cannot be Rezzed with my banner. Fenryl is the one rezzing Kazzune. Seems weird, i can rez other people and his other toons recieve rezzes.
